rpc/trace.go

```diff
@@ -18,6 +18,7 @@ func adaptBlockTrace(block *BlockWithTxs, blockTrace *starknet.BlockTrace, legac
 	}
 	traces := make([]TracedBlockTransaction, 0, len(blockTrace.Traces))
 	for index := range blockTrace.Traces {
+		// why not index, feederTrace := range above ?
```

Comment:
I belive that would work too, I might have wanted to avoid having traces copied on each iteration.